receiver/otlpreceiver/erroradapter.go

```diff
@@ -0,0 +1,48 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package otlpreceiver // import "go.opentelemetry.io/collector/receiver/otlpreceiver"
+
+import (
+	"net/http"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func gRPCToHTTP(s *status.Status) int {
```

Comment:
Is there an official guideline around translation from one protocol to another?

Reply:
Here it is https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto